### PR TITLE
Multi Player Manic Miner

### PR DESCRIPTION
**Pull request by monkeyx-net**
new file:   ports/multimanicminer/README.md
	new file:   ports/multimanicminer/multimanicminer/conf/pulse/cookie
	new file:   ports/multimanicminer/multimanicminer/multimanicminer.gptk
	new file:   ports/multimanicminer/multimanicminer/multimanicminer.pck
	new file:   ports/multimanicminer/multimanicminer/multimanicminer_LICENSE.txt
	new file:   ports/multimanicminer/port.json
	new file:   ports/multimanicminer/screenshot.png

# New Port for Multi Players Manic Miner

## Game Information
- **Title**: Multi Players Manic Miner
- **URL**: https://github.com/SteveSmith16384/ManicMinerMultiplayer

### CFW Tests
Ensure your game has been tested on all major CFWs:
- [ x] AmberELEC
- [x ] ArkOS
- [x ] JELOS

### Resolution Tests
Test all major resolutions:
- [ x] 480x320
- [ x] 640x480
- [ x] 720x720 (RGB30)
- [ x] Higher resolutions (e.g., 1280x720)

## File Structure
- Your port should have the following structure:
  - portname/
    - port.json
    - README.md
    - screenshot.jpg
    - cover.jpg
    - Port Name.sh
    - portname/
      - <portfiles here>

## Additional Resources
For an in-depth guide on creating a pull request, refer to: [PortMaster Game Packaging Guide](https://portmaster.games/packaging.html#creating-a-pull-request)

### CODE REVIEW

**Review by Cebion — DISMISSED** [review]
👍

File: ports/multimanicminer/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+## Notes
+
+This is a  recreation of the first level, the Central Cavern, of Manic Miner, except it allows up to 4 players to play simultaneously on the same PC.
+
+It's a race to be the first to collect all the keys and then get back to the flashing waffle-type thing, using either gamepads or the keyboard.
+
+Thanks to [Steven Smith]https://github.com/SteveSmith16384/ManicMinerMultiplayer for this game.
```
Comment (kloptops) [inline ports/multimanicminer/README.md:7]: Markdown is wrong. it should be

```markdown
Thanks to [Steven Smith](https://github.com/SteveSmith16384/ManicMinerMultiplayer) for this 
```

**Review by kloptops — CHANGES_REQUESTED** [review]
Markdown needs a fix.

**Comment by monkeyx-net** [comment]
> Markdown needs a fix.

Hopefully I managed to do this on my phone.

**Review by kloptops — APPROVED** [review]
👍

**Review by Cebion — APPROVED** [review]

**Pull request merged**